.cirrus.yml

```
@@ -0,0 +1,119 @@
+env:
+  CPPFLAGS: -I/usr/local/include
+  LDFLAGS: -L/usr/local/lib
+  PREFIX: ${CIRRUS_WORKING_DIR}/project
+  PATH: ${PREFIX}/bin:${PATH}
+  BIRCH_FLAGS: '--unit=dir --arch=native --jobs=2'
+  MODE_FLAGS: '--disable-debug --enable-test --enable-release'
```

```suggestion
  MODE_FLAGS: '--enable-debug'
```
This should speed up build time a lot: it only compiles one version (debug) and not two versions (test, used for code coverage, and release, used for the more extensive tests) as in the CircleCI tests.

---

...this because you mentioned you've only enabled the smoke tests, and looks like CodeCov isn't here (doesn't need to be).

---

Edited to explicitly `--disable-test --disable-release` even though they're off by default, to keep the configuration explicit for maintainers.